test/multiverse/suites/ruby_openai/ruby_openai_instrumentation_test.rb

```diff
@@ -158,7 +230,7 @@ def test_llm_custom_attributes_added_to_embedding_events
         'triceratops' => 'herbivore'
       })
       stub_post_request do
-        client.embeddings(parameters: chat_params)
+        client.embeddings(parameters: embeddings_params)
```

Comment:
This test is ensuring `llm.` custom events are added. This change doesn't affect the test, but it always should've been `embeddings_params`.